apm-sniffer/optional-plugins/optional-spring-plugins/spring-webflux-5.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/spring/webflux/v5/DispatcherHandlerHandleMethodInterceptor.java

```diff
@@ -77,13 +77,14 @@ public void beforeMethod(EnhancedInstance objInst, Method method, Object[] allAr
         exchange.getAttributes().put("SKYWALING_SPAN", span);
     }
     
-    private void setPattern(AbstractSpan span, ServerWebExchange exchange) {
+    private void maybeSetPattern(AbstractSpan span, ServerWebExchange exchange) {
         if (span != null) {
-            Object pathPattern = exchange.getAttribute(HandlerMapping.BEST_MATCHING_PATTERN_ATTRIBUTE);
-            if (pathPattern != null) {
-                span.setOperationName(((PathPattern) pathPattern).getPatternString());
+            PathPattern pathPattern = exchange.getAttribute(HandlerMapping.BEST_MATCHING_PATTERN_ATTRIBUTE);
+            if (pathPattern != null && pathPattern.matches(exchange.getRequest().getPath().pathWithinApplication())) {
```

Comment:
What does `pathPattern.matches(exchange.getRequest().getPath().pathWithinApplication())` mean? Could you add comments about this?